src/Api/Payments/ReservationOfFixedAmount.php

```diff
@@ -236,14 +298,27 @@ protected function configureOptions(OptionsResolver $resolver)
      * @param Request           $request
      * @param ResponseInterface $response
      *
-     * @return ReservationOfFixedAmountResponse
+     * @return PaymentRequestResponse
```

Comment:
This will break the current integration the merchants have. Is it necessary to change the response type?

See my other comments.

Reply:
Updated